superset/assets/src/dashboard/components/gridComponents/Chart.jsx

```diff
@@ -169,6 +172,14 @@ class Chart extends React.Component {
     this.props.changeFilter(this.props.chart.id, newSelectedValues);
   }
 
+  handleFilterMenuOpen(chartId, column) {
+    this.props.setFocusedFilterField(chartId, column);
+  }
+
+  handleFilterMenuClose() {
+    this.props.setFocusedFilterField();
```

Comment:
not sure if this is super clear, might prefer `unsetFocusedFilterField` or perhaps `setFocusedFilterField(null)`?

Reply:
the purpose of empty parameter here is to clear/unset focused filter field state. 
I thought we prefer no params than explicitly `null` params? I rarely see any example in our code that explicitly pass a `null` to function.

Reply:
updated. created `unsetFocusedFilterField` action creator.